To: vim_dev@googlegroups.com
Subject: Patch 7.3.345
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.345
Problem:    When switching language with ":lang" the window title doesn't
	    change until later.
Solution:   Update the window title right away. (Dominique Pelle)
Files:	    src/ex_cmds2.c


*** ../vim-7.3.344/src/ex_cmds2.c	2011-06-26 04:25:24.000000000 +0200
--- src/ex_cmds2.c	2011-10-20 21:31:09.000000000 +0200
***************
*** 4154,4159 ****
--- 4154,4162 ----
  	    /* Set v:lang, v:lc_time and v:ctype to the final result. */
  	    set_lang_var();
  # endif
+ # ifdef FEAT_TITLE
+ 	    maketitle();
+ # endif
  	}
      }
  }
*** ../vim-7.3.344/src/version.c	2011-10-20 21:27:57.000000000 +0200
--- src/version.c	2011-10-20 21:38:32.000000000 +0200
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     345,
  /**/

-- 
You can't have everything.  Where would you put it?
		-- Steven Wright

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
